static/app/components/feedback/list/mailboxPicker.tsx

```diff
@@ -21,6 +24,9 @@ export default function MailboxPicker({onChange, value}: Props) {
       >
         <SegmentedControl.Item key="unresolved">{t('Inbox')}</SegmentedControl.Item>
         <SegmentedControl.Item key="resolved">{t('Resolved')}</SegmentedControl.Item>
+        <SegmentedControl.Item key="ignored" disabled={!hasSpamFeature}>
+          {t('Spam')}
+        </SegmentedControl.Item>
```

Comment:
Can we add a tooltip explaining why it's disabled?

Reply:
meh, i'll put some effort into removing the button first. I'll take a look in a bit to help